packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -216,7 +216,7 @@ export const BaseInput = ({
           handleOnChange={handleOnChange}
         />
       </Box>
-      <Box marginLeft={isLabelLeftPositioned ? 120 : 'auto'}>
+      <Box marginLeft={isLabelLeftPositioned ? 136 : 0}>
```

Comment:
😢 136 this magic numbers gonna bite us one day. 

Let atleast add a comment? or put this in a named variable that this represents 120+16px

Reply:
fixed in e263fd9b727b4b26f7de680d3ce34de705cca296